examples/example-gateway/clients/contacts/fixture/fixture.go

```diff
@@ -0,0 +1,23 @@
+package fixture
+
+import (
+	mc "github.com/uber/zanzibar/examples/example-gateway/build/clients/contacts/mock-client"
+	gen "github.com/uber/zanzibar/examples/example-gateway/build/gen-code/clients/contacts/contacts"
+)
+
+var saveContactsFixtures = map[string]*mc.SaveContactsFixture{
+	"success": {
```

Comment:
Convert from `map[string]*mc.SaveContactsFixture` to struct defined in generated code but initialized in user code. This will reduce time to detect the error to build time (field is not present) rather than test time (null pointer exception).

Reply:
Great suggestion. Replaced the map with struct now.